docs/source/fitting/fitfunctions/AsymmetricPearsonVII.rst

```diff
@@ -0,0 +1,81 @@
+.. _func-AsymmetricPearsonVII:
+
+======================
+Asymmetric Pearson VII
+======================
+
+.. index:: AsymmetricPearsonVII
+
+Description
+-----------
+
+The asymmetric Pearson VII function (sometimes it is also referred to as the split-Pearson VII function) is a function that combines
+two Pearson VII distributions so to fit sharp peak curves. It is useful for analysis of X-ray diffraction peaks which consist of two separable components.
+
+In one of the representations, the asymmetric Pearson VII function :math:`A(x)` can be written as a superposition of the "left" and "right" Pearson VII functions :math:`P_L(x)` and :math:`P_R(x)`,
+
+.. math:: A(x) = P_L(x) \, \theta(x_0 - x) + P_R(x) \, \theta(x - x_0),
```

Comment:
could this be written as:

A(x) = P_L(x)\theta(x_0-x) + P_R(x)(1-\theta(x-x0))

Reply:
You probably mean that A(x) = P_L(x)\theta(x_0-x) + P_R(x)(1-\theta(x_0-x)), right? If that's the case, then yes, this would be an alternative way to write A(x). I just included this expression in the documentation

Reply:
yes, thank you